Makefile

```diff
@@ -57,6 +61,21 @@ lint: fmt vet generate ## All-in-one linting
 generate: ## Generate additional code and artifacts
 	@go generate ./...
 
+.PHONY: install-crd
+install-crd: export KUBECONFIG = $(KIND_KUBECONFIG)
+install-crd: generate ## Install CRDs into cluster
```

Comment:
`setup-kind` is missing. I assume if it's export `KIND_KUBECONFIG` that it should also prepare the cluster?

Reply:
generally yes